js/index.js

```diff
@@ -1,7 +1,21 @@
 // Iteration 1: Names and Input
+let hacker1 = "Viktoria";
+console.log (`"The driver's name is ${hacker1}."`);
+let hacker2 = "Rich";
+console.log (`"The navigator's name is ${hacker2}."`);
+
+
```

Comment:
Please mind the blank lines here. If they do not serve a purpose, they can be removed.